src/views/BuildingView.vue

```diff
@@ -1,10 +1,9 @@
 <template>
-  <div class="home">
+  <div>
     <building-list :locationName=location></building-list>
```

Comment:
müsste hier nicht noch "" rum?
```suggestion
    <building-list :locationName="location"></building-list>
```